packages/rest/src/lib/REST.ts

```diff
@@ -11,7 +11,7 @@ import {
 	type RequestData,
 	type RouteLike,
 } from './RequestManager.js';
-import type { IHandler } from './handlers/IHandler.js';
+import type { IHandler } from './handlers/Handler.interface.js';
```

Comment:
p sure I remember Crawl saying he's not a fan of doing `.interface.ts`

Reply:
Yes I have to concur here, this pattern was only really used in the docgen, and that code is anything but what we should move forward with.

Reply:
using .d.ts instead!